File: libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationTimeFormat.java
package com.mapbox.services.android.navigation.v5.navigation;

import androidx.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

public class NavigationTimeFormat {

  @Retention(RetentionPolicy.SOURCE)
  @IntDef( {NONE_SPECIFIED, TWELVE_HOURS, TWENTY_FOUR_HOURS})
  public @interface Type {
  }

  public static final int NONE_SPECIFIED = -1;
  public static final int TWELVE_HOURS = 0;
  public static final int TWENTY_FOUR_HOURS = 1;
}

File: libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationTimeFormat.kt
@file:JvmName("NavigationTimeFormatKt")

package com.mapbox.services.android.navigation.v5.navigation

import androidx.annotation.IntDef

@Retention(AnnotationRetention.RUNTIME)
@IntDef(NONE_SPECIFIED, TWELVE_HOURS, TWENTY_FOUR_HOURS)
annotation class TimeFormatType

const val NONE_SPECIFIED = -1
const val TWELVE_HOURS = 0
const val TWENTY_FOUR_HOURS = 1
